pkg/clusterversion/cockroach_versions.go

```diff
@@ -517,53 +521,57 @@ var versionsSingleton = keyedVersions{
 		Version: roachpb.Version{Major: 22, Minor: 1, Internal: 4},
 	},
 	{
-		Key:     EnsurePebbleFormatVersionRangeKeys,
+		Key:     PebbleFormatSplitUserKeysMarkedCompacted,
 		Version: roachpb.Version{Major: 22, Minor: 1, Internal: 6},
 	},
 	{
-		Key:     EnablePebbleFormatVersionRangeKeys,
+		Key:     EnsurePebbleFormatVersionRangeKeys,
```

Comment:
I shifted everything down to slot in the `PebbleFormatSplitUserKeysMarkedCompacted` version before the range keys version. I _think_ that's fine?